packages/mockyeah-cli/bin/mockyeah-play.js

```diff
@@ -14,26 +14,36 @@ const boot = require('../lib/boot');
 const inquirer = require('inquirer');
 const chalk = require('chalk');
 const tildify = require('tildify');
-let name;
+const request = require('request');
 
-program
-  .option('-v, --verbose', 'Verbose output')
-  .parse(process.argv);
+program.option('-v, --verbose', 'Verbose output').parse(process.argv);
+
+const withName = (env, name) => {
+  const { adminUrl } = env;
+
+  request.get(`${adminUrl}/play?name=${name}`, (err, res) => {
+    if (err) {
+      // TODO: Detect errors that shouldn't result in local fallback.
+      require(env.modulePath).play(name);
+    }
+  });
```

Comment:
This wasn't straight forward to me at first, but it makes sense now. It might have been easier to understand if `withName` was `playRemoteOrLocal`... or something to that affect.